DigitalLearningSolutions.Data.Tests/Services/SectionContentServiceTests.cs

```diff
@@ -49,10 +49,10 @@ public void Get_section_content_should_return_section_content()
             expectedSectionContent.Tutorials.AddRange(
                 new[]
                 {
-                    new SectionTutorial("Introduction to applications", 0, "Not started", 0, 17, 1461),
-                    new SectionTutorial("Common screen elements", 0, "Not started", 0, 11, 1462),
-                    new SectionTutorial("Using ribbon tabs", 0, "Not started", 0, 6, 1463),
-                    new SectionTutorial("Getting help", 0, "Not started", 0, 11, 1464)
+                    new SectionTutorial("Introduction to applications", 0, "Not started", 0, 17, 1461, true),
+                    new SectionTutorial("Common screen elements", 0, "Not started", 0, 11, 1462, true),
+                    new SectionTutorial("Using ribbon tabs", 0, "Not started", 0, 6, 1463, true),
+                    new SectionTutorial("Getting help", 0, "Not started", 0, 11, 1464, true)
```

Comment:
As discussed could you add tests making sure we're returning the same tutorials as the old stored procedure, like the test we have for the initial menu page? We should test a few customisations for different users. In particular we should definitely test the courses which had this issue (customisation ids are on the ticket) 🙂